### DISCUSSION TITLE
I can't log in to Versel through a git account

### DISCUSSION THREAD

**Discussion by nataligavryliak** [vercel/community#3669] [Account Issue] [answered]
### Question

I can't log in to Versel through a git account. Vercel says that the account was not found. Although the git account is created
nataligavryliak (git username)

**Comment by amyegan** [comment, marked as answer]
Hey @nataligavryliak. I can't find an account with that username. Please visit https://vercel.com/signup to create your Vercel account.

If you continue to have trouble, please email [registration@vercel.com](mailto:registration@vercel.com) for more help. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Comment by nataligavryliak** [comment]
When trying to go to Versel, it gives out
 This user account is blocked.  vercelusername іs: nataligavryliak

**Reply by codebymd** [reply]
did you already fixed yours? I have the same problem and I can't fixed mine this is prosrating :(
